src/chrome/content/rules/iplocation.net.xml

```diff
@@ -0,0 +1,23 @@
+<!--
+
+	Non-functional subdomain:
+		- forums	(display home page)
+
+-->
+
+<ruleset name="iplocation.net">
+
+	<target host=    "iplocation.net" />
+	<target host="www.iplocation.net" />
+
+  	<securecookie host="^www\.iplocation\.net$" name=".*" />
+
+	<rule from="^http://iplocation\.net/"
+		to="https://www.iplocation.net/" />
```

Comment:
Why redirect to www? ^iplocation.net seems to work fine. If there's no problem, please don't preempt the server's redirect. Doing so only gives an adversary on the wire greater confidence that the user is using https-everywhere.